aws/resource_aws_ecs_service_test.go

```diff
@@ -730,17 +830,18 @@ resource "aws_ecs_service" "main" {
     subnets = ["${aws_subnet.main.*.id}"]
   }
 }
-`, rInt, rInt, rInt, rInt)
+`, sg1Name, sg2Name, clusterName, tdName, svcName)
 }
 
-func testAccAWSEcsService_healthCheckGracePeriodSeconds(rName string, testName string, healthCheckGracePeriodSeconds int) string {
+func testAccAWSEcsService_healthCheckGracePeriodSeconds(vpcNameTag, clusterName, tdName, roleName, policyName,
+	tgName, lbName, svcName string, healthCheckGracePeriodSeconds int) string {
 	return fmt.Sprintf(`
 data "aws_availability_zones" "available" {}
 
 resource "aws_vpc" "main" {
   cidr_block = "10.10.0.0/16"
   tags {
-    Name = "%[2]s"
```

Comment:
We should probably decide a good number of args cutoff for using the positional args for `fmt.Sprintf()`. I personally find the positional args helpful when dealing with 3+ although I know they aren't always linted correctly with typing. In this case they're removed here but added in `tpl_testAccAWSEcsService_withLbChanges`

Ideally though maybe we can come up with clever ways to create reusable templates across resource tests. In this case it'd be great if we could just reference something like `testAccAWSVpc_basic(name)` to DRY up the testing templates, especially since we need to reference `aws_vpc` in a bunch of places.